net-irc/atheme-services/atheme-services-7.2.9.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils flag-o-matic perl-module user
+
+DESCRIPTION="A portable and secure set of open-source and modular IRC services"
+HOMEPAGE="https://github.com/atheme/atheme"
+SRC_URI="https://github.com/atheme/atheme/releases/download/v${PV}/atheme-${PV}.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd ~amd64-linux"
+IUSE="cracklib largenet ldap nls +pcre perl profile ssl"
+
+RDEPEND=">=dev-libs/libmowgli-2.1.0:2
+	cracklib? ( sys-libs/cracklib )
+	ldap? ( net-nds/openldap )
+	nls? ( sys-devel/gettext )
+	perl? ( dev-lang/perl )
+	pcre? ( dev-libs/libpcre )
+	ssl? ( dev-libs/openssl:* )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+pkg_setup() {
+	# the dependency calculation puts all of the .c files together and
+	# overwhelms cc1 with this flag :-(
+	filter-flags -combine
+
+	if use profile; then
+		# bug #371119
+		ewarn "USE=\"profile\" is incompatible with the hardened profile's -pie flag."
+		ewarn "Disabling PIE. Please ignore any warning messages about -nopie being invalid."
+		append-flags -nopie
+	fi
+
+	enewgroup ${PN}
+	enewuser ${PN} -1 -1 /var/lib/atheme ${PN}
+}
+
+src_unpack() {
+	default
+	# Fix source dir.
+	mv "${WORKDIR}/atheme-${PV}" "${WORKDIR}/${P}"
+}
+
+src_prepare() {
+	# fix docdir
+	sed -i -e 's/\(^DOCDIR.*=.\)@DOCDIR@/\1@docdir@/' extra.mk.in || die
```

Comment:
Any clue if this is still necessary? The problem with `sed`s like this is that for every bump you have to verify whether it still applies since it will not report not doing anything in any way. Would be nice to figure this out, and possibly submit a proper patch upstream.